bananalyzer/__main__.py

```diff
@@ -129,10 +129,10 @@ def parse_args() -> Args:
         help="A list of ids to skip tests on, separated by commas",
     )
     parser.add_argument(
-        "-q",
-        "--quiet",
+        "-v",
+        "--verbose",
```

Comment:
can we also add `--v`? to match what pytest does

Reply:
nah